views/templates/block/checklist.twig

```diff
@@ -59,7 +59,7 @@
                     </tr>
                 {% endif %}
                 <tr>
-                    <td>{{ 'PHP\'s "Safe mode" option is turned off'|trans({}, 'Modules.Autoupgrade.Admin') }}</td>
+                    <td>{{ 'PHP\'s "Safe mode" option is turned off'|trans({}, 'Modules.Autoupgrade.Admin')|raw }}</td>
```

Comment:
yes, I'm not sure what's going wrong, but the bug appeared when we removed the raw, so for the sake of the release I put back the code template "as is", since this weird behavior is not a regression, I'm creating an issue about this weird behavior for after the release though

![failed-step-1](https://user-images.githubusercontent.com/1784781/192319933-a26fdd41-bcb6-496d-901b-e8d2376ecc2b.png)

Reply:
I don't understand why either 🤔 but ok for the quickfix